notebooks/vjepa2_demo.py

```diff
@@ -63,14 +63,14 @@ def get_video():
     return video
 
 
-def forward_vjepa_video(model_hf, model_pt, hf_transform, pt_transform):
+def forward_vjepa_video(model_hf, model_pt, hf_transform, pt_transform, device):
```

Comment:
This changes the default function signature. Can you modify it so that the default device is `cuda`, as was the case with the original function signature?